internal/cloudapi/openapi.yml

```diff
@@ -243,6 +253,36 @@ components:
           example: ['123456789012']
           items:
             type: string
+    GCPUploadRequestOptions:
+      type: object
+      required:
+        - bucket
```

Comment:
Just something that I've encountered right now: Can you document what happens if you don't pass the individual optional values?

Reply:
I added description to each value with explanation of what will happen.